docs/manuals/gui/configuration.md

```diff
@@ -33,92 +33,102 @@ method of your `Gui` instance using the _kwargs_ parameter.
+   - *notification_duration* (int, default: 3000): the time, in milliseconds, that
+     notifications should remain visible (see [Notifications](notifications.md) for
+     details).
+   - *debug* (bool, default: True): set to True if you want to be provided with detailed
      debugging information messages from the server.<br/>
-     You can force the _debug_ mode using the _--debug_ option when launching
+     If the debug mode is set, then the *async_mode* parameter of the call to `Gui.run()^`
+     is overridden to "threading" to use the Flask built-in development server.<br/>
+     To avoid warning from the websocket layer when using the *debug* mode, make sure
+     you install the *simple-websocket* optional package.<br/>
+     You can force the *debug* mode using the *--debug* option when launching
      your application.<br/>
-     Or you can force **not** to use the _debug_ mode using the _--no_debug_ option
+     Or you can force **not** to use the *debug* mode using the *--no_debug* option
      when launching your application.
-   - _flask_log_ (bool, default: False): if set to True, you can get a full, real-time
+   - *flask_log* (bool, default: False): if set to True, you can get a full, real-time
      log from the Flask server. This may be useful when trying the find the reason why
      a request does not behave as expected.
-   - _use_reloader_ (bool, default: True): If True, the application watches its Python
+   - *use_reloader* (bool, default: True): If True, the application watches its Python
      source file while running, and reloads the entire script should the file be
      modified. If False, there is no such watch in place.<br/>
-     You can force the _use_reloader_ mode using the _--use-reloader_ option when
+     You can force the *use_reloader* mode using the *--use-reloader* command line when
      launching your application.<br/>
-     Or you can force **not** to use the _use_reloader_ mode using the _--no-reloader_
-     option when launching your application.
-   - _single_client_ (bool, default: False): set to True if only a single client can connect.
-     False indicates that multiple clients can connect to the server.
-   - _propagate_ (bool, default: True): the default value that will be used for every
-     _propagate_ property value, for all controls. Please look at the section on the
-     [_propagate_ property](viselements/#the-propagate-property) for details).
-   - _time_zone_ (str, default: "client"): indicates how date and time values should be
+     Or you can force **not** to use the *use_reloader* mode using the *--no-reloader*
+     command line option when launching your application.<br/>
+     Note that the reloading only takes place if either the *async_mode* parameter of
+     `Gui.run()^` is set to "threading" or the *debug* parameter is set to True.
+   - *single_client* (bool, default: False): set to True if only a single client can
```

Comment:
Multiple clients can connect but will share the same state ...

Reply:
I going to hide this detail from users... Thx